extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -607,7 +615,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
   }
 
   /** @override */
-  getAdUrl(consentState, opt_rtcResponsesPromise) {
+  getAdUrl({consentState, consentString}, opt_rtcResponsesPromise) {
```

Comment:
Don't see why you need to deconstruct the consent object given you're just rebuilding it later.... leave it as the tuple

Reply:
The function wants to access the `consentState` member individually here:
```
if (
      consentState == CONSENT_POLICY_STATE.UNKNOWN &&
      this.element.getAttribute('data-npa-on-unknown-consent') != 'true'
    )
```
But sure, accessing through the object is fine. Modified.